File: C/3 Seleção/final.c
#include <stdio.h>

int main(){
    float n1, n2;
    scanf("%f %f", &n1, &n2);

    float media = (n1+n2)/2;

    if(media>=7) printf("aprovado\n");
    else if(media<4) printf("reprovado\n");
    else{
        float af;
        scanf("%f", &af);

        media = (media+af)/2;
        if(media>=5)printf("aprovado na final\n");
        else printf("reprovado na final\n");
    }
}

File: Fup is Fun/Python/3 Seleção/final.py
n1 = float(input())
n2 = float(input())
media = (n1+n2)/2

if media>=7: print("aprovado")
elif media<4: print("reprovado")
else:
    af = float(input())
    media = (media+af)/2
    if media>=5: print("aprovado na final")
    else: print("reprovado na final")